src/Client.php

```diff
@@ -57,33 +55,34 @@ function (RequestInterface $request) {
         $this->authenticator = $authenticatorFactory->getAuthenticator($clientFactory, 'https://vault.azure.net');
     }
 
-    private function sendRequest(Request $request)
+    private function sendRequest(Request $request): array
     {
         try {
-            if (!$this->token) {
+            if (empty($this->token)) {
```

Comment:
`if ($this->token === '')`? Je to natypovany na `string`, takze nikdy nebude `null`, int apod.

Reply:
typed property must not be accessed before initialization

Reply:
aha, tak by to chtelo mu dat vychozi `null`? :smile: